services/ResumeService.ts

```diff
@@ -0,0 +1,23 @@
+import { ResumeModel } from "models/ResumeModel";
+import Repositories, { TransactionsManager } from "repositories";
+import { NotFoundError } from "routing-controllers";
+import { Service } from "typedi";
+import { EntityManager } from "typeorm";
+import { InjectManager } from "typeorm-typedi-extensions";
+import { ResumeRepository } from "../repositories/ResumeRepository";
+
+@Service()
+export default class ResumeService {  
+  private transactions: TransactionsManager;
+
+  constructor(@InjectManager() entityManager: EntityManager) {
+    this.transactions = new TransactionsManager(entityManager);
+  }
+
+  public async getVisibleResumes() : Promise<ResumeModel[]> {
+    const resumes = await this.transactions.readOnly(async (txn) => Repositories
+    .resume(txn).findVisible());
+    if (!resumes) throw new NotFoundError('Resumes was not found');
```

Comment:
nit: formatting - line 19 should be tabbed inwards

Reply:
also, do we need to throw an error if no resumes were found? I imagine just returning an empty array should suffice, and thus no resumes would be displayed on the frontend.

Reply:
So far I think an empty array should suffice